app/components/form/instance.hbs

```diff
@@ -15,7 +15,7 @@
             @forceShowErrors={{true}}
             @useNewListingLayout={{true}}
             @level={{2}}
-            class="au-u-max-width-medium"
+            class="au-u-max-width-medium {{if this.isEditable '' 'disabled'}}"
```

Comment:
I find this ternary if here a bit confusing and it doesn't seem necessary, I would suggest using 
`{{unless this.isEditable 'disabled'}}`

Reply:
Oh didn't know you could do that, thanks!